### PR TITLE
axe.configure disableOtherRules option

### LINKED ISSUE #688: Add a disableOtherRules option to axe.configure
This will help get rid of some confusion about why rules are running after you called axe.configure with a new set of rules.

### PR DESCRIPTION
**Pull request by WilcoFiers**
Closes #688

### CODE REVIEW

**Comment by WilcoFiers** [comment]
@marcysutton I don't think integration tests are good for that. They are very slow because they need to load in a separate page. What exactly do you think isn't covered in the tests that you'd want to see covered?

**Comment by marcysutton** [comment]
I mentioned integration tests because the core unit test relies on non-public API methods and properties, like `axe._load` and `axe._audit`. I always like to know that the whole system is working, not just the internal stuff.

**Comment by WilcoFiers** [comment]
@marcysutton Updated, please review again.

File: test/integration/full/configure-options/configure-options.js [hunk 1]
```diff
@@ -1,50 +1,82 @@
-describe('Check Configure Options', function() {
+describe('Configure Options', function() {
```
Comment (marcysutton) [inline test/integration/full/configure-options/configure-options.js]: I named it `Check configure options` to distinguish between the other `options-parameter` tests in the same folder. Perhaps these new tests should be combined with those instead? Or all together?

Comment (WilcoFiers) [reply test/integration/full/configure-options/configure-options.js]: oh! I see, you're right. I put them in the wrong place. My bad.

Comment (WilcoFiers) [reply test/integration/full/configure-options/configure-options.js]: Actually, no, this is correct. The `options parameter` is the `axe.run` options. This is for `axe.configure`. Your existing test was for `axe.configure({ check })`, this new one is `axe.configure({ disableOtherRules })`.

**Pull request merged**